File: go/app/src/rabbitmq/queues.go
package rabbitmq

const (
	ExchangeName      = "go"
	QueueGolang       = "go"
	QueueSystemGolang = "system-go"
)

// GetQueueName возвращает название очереди в зависимости от типа события
func GetQueueName(isSystem bool) string {
	if isSystem {
		return QueueSystemGolang
	}
	return QueueGolang
}

File: rust_actix/app/src/rabbitmq/queues.rs
pub const EXCHANGE_NAME: &str = "rust-actix";
pub const QUEUE_ACTIX: &str = "rust-actix";
pub const QUEUE_SYSTEM_ACTIX: &str = "system-rust-actix";

pub fn get_queue_name(is_system: bool) -> &'static str {
    if is_system {
        QUEUE_SYSTEM_ACTIX
    } else {
        QUEUE_ACTIX
    }
}
